### PR TITLE
Rn entry 2023.06.0

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** June Release Note draft
- **Is this PR related to a Monolith release?** 2023.06.0

### Resolves:

* [OKTA-617748](https://oktainc.atlassian.net/browse/OKTA-617748)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 1]
```diff
@@ -4,15 +4,89 @@ title: Okta Identity Engine API Products release notes 2023
...
+
+#### Multibrand customizations are GA in Production
+
+Multibrand customizations allow customers to use one org to manage multiple brands and multiple custom domains. This drastically simplifies multi-tenant architectures where customers create multiple orgs to satisfy branding requirements. Multibrand customizations allow orgs to create up to three custom domains (more upon request), which can be mapped to multiple sign-in pages, multiple sets of emails, error pages, and multiple versions of the End-User Dashboard. See [Brands](/docs/concepts/brands/).  <!-- OKTA-568807 -->
+
+#### Pagination for the Brands API is GA in Production
+
+The Brands API now supports [pagination](/docs/reference/core-okta-api/#pagination) when returning lists of brands. Previously, users would get a list of all brands in the org. With pagination, users receive 20 records per page. See [Customizations](/docs/api/openapi/okta-management/management/tag/Customization). <!-- OKTA-574011 -->
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:36]:
```suggestion
The Brands API now supports [pagination](/docs/reference/core-okta-api/#pagination) when returning lists of brands. Previously, users would get a list of all brands in the org. With pagination, users receive 20 records per page. See [Customizations](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/). <!-- OKTA-574011 -->
```

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 2]
```diff
@@ -4,15 +4,89 @@ title: Okta Identity Engine API Products release notes 2023
...
+
+#### Policy Simulation API is EA in Preview
+
+With the Policy API `/simulate` endpoint, you can quickly and easily test policies and validate whether your desired security outcomes will be achieved. This endpoint allows you to simulate user access attributes, such as IP address, device, risk, and so on, to test whether the user will be granted access to the specified application. This endpoint is implemented in the Admin Console as the [Access Testing Tool](https://help.okta.com/okta_help.htm?type=oie&id=csh-access-testing-tool). The`/simulate` endpoint helps you identify potential security risks and compliance issues before you implement a policy. See [Policy API](/docs/reference/api/policy/#policy-simulation-operations) and [Test your policies with access simulations](/docs/guides/). <!--OKTA-593826-->
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:62]: Are the updates to the Policy API not available in redocly?

**Review by brentschaus-okta — APPROVED** [review]
LGTM!

A few URLs that need fixing, and a question.

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md (hunk 2 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:62]: They are. We've updated both locations. Not sure where to point? Or should we do both?

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md (hunk 1 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:36]: Thanks, good catch.

**Pull request merged**